### ISSUE TITLE
Remove link to login for tokens

### ISSUE THREAD

**Issue by adam3smith** [QualitativeDataRepository/AnnoREP-Frontend#43]
![image](https://user-images.githubusercontent.com/128986/156244909-fcb9ca3c-9726-4fd5-9f7e-5828efb012c8.png)

In testing, users click on the first link & are confused. Trying to get a token routes users through login anyway.

**Comment by adam3smith** [comment]
@tohuynh -- depending on how realistic 0Autho for Hypothesis looks, could you revert the Hypothesis part for this, i.e. revert back to "[Log in to Hypothes.is](https://hypothes.is/login), then get your [Hypothes.is API token here](https://hypothes.is/account/developer)

**Comment by tohuynh** [comment]
@adam3smith At first glance, OAuth seems realistic. I need to read more documentation and test out if it's possible to combine password (API token for us) login + OAuth login in NextAuth.js to specify that the user is logged in to AnnoRep after they do both. I think it's possible to either log in with an API token or a custom OAuth provider like Hypothesis -- i.e. users choose to login to AnnoREP with QDR API token or Hypothesis OAuth (but this is not what AnnoREP needs).

Regardless, I will add back the Hypothesis login link.

**Comment by tohuynh** [comment]
@adam3smith Given the current tech stack it's not currently feasible to add Hypothesis OAuth for login.

Currently, AnnoRep uses a credentials provider to create a JWT (JSON web token to store user's credentials) and then create a session upon successful login. Another provider like Hypothesis OAuth can't be combined with the credentials provider to create one JWT. What actually happens is if the user logins with credentials provider, and then login with Hypothesis OAuth provider, the first JWT is replaced with a new JWT (and removes any session info about the user's QDR API token). The vice versa is also true.

See [this disccussion](https://github.com/nextauthjs/next-auth/discussions/3157).

We could use a DB to persist users, accounts, and sessions, and then combine two providers so that both the QDR api token and Hypothesis access token are available in one session. But unfortunately, the credentials provider must use JWT to persist user's data. (And AnnoRep needs the credentials provider for the QDR API token login).

In summary, it looks like Hypothesis OAuth is not currently possible.
